packages/app/src/app/location-selection/selection-criteria/selection-criteria.component.ts

```diff
@@ -407,19 +411,30 @@ export class SelectionCriteriaComponent {
 
     // console.log('criteria before/after', formValue.criteria, criteria);
 
-    let siteSuitability: SiteSuitabilityCriteria | undefined = undefined;
+    const sharedCriteria: SharedCriteria = {
```

Comment:
be more explicit about starting with shared criteria and creating payloads from that